### ISSUE TITLE
[bitnami/mysql] initdbScriptsConfigMap find: '/docker-entrypoint-startdb.d/': No such file or directory

### ISSUE THREAD

**Issue by doznap** [bitnami/charts#17718]
### Name and Version

bitnami/mysql 9.7.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

kubernetes.
1 create a configmap from a mysql dump
```
./kubectl create configmap mysql-scheme --from-file=initdb.sql  || echo "configmap already exists"
```
2 use the helm options to install the mysql chart
```
helm upgrade -i mysql bitnami/mysql \
            --set auth.rootPassword=$MYSQL_DB_ROOT_PASS \
            --set auth.username=$MYSQL_DB_USER \
            --set auth.password=$MYSQL_DB_USER_PASS \
            --set auth.database=$MYSQL_DB_NAME \
            --set initdbScriptsConfigMap="mysql-scheme" \
            --version 9.7.1
```            

### Are you using any custom parameters or values?

yes written above ^^

### What is the expected behavior?

the mysql initialization should apply the config map sql script to the database creating tables

### What do you see instead?
```
mysql 21:29:17.05 INFO  ==> Starting mysql in background
find: '/docker-entrypoint-startdb.d/': No such file or directory
mysql 21:29:33.16 INFO  ==> Stopping mysql
```

### Additional information

I'm not sure if this is an issue or i'm doing something incorrectly...
so if there is a better place for me to ask questions to the community, please let me know, 
and thank u!

**Comment by maxsargentdev** [comment]
I am also experiencing this, not sure on the exact root cause but sure enough /docker-entrypoint-startdb.d is missing from the image when I shell into the running kubernetes pod.

I mounted an empty volume to this location, this removed the error message but it did not lead to the initialization scripts being run correctly so there must be another issue.

**Comment by migruiz4** [comment]
Hi @doznap @maxsargentdev,

Please notice that the Bitnami mysql image has two scripts mountpoints:
-  `/docker-entrypoint-initdb.d`
- `/docker-entrypoint-startdb.d`

The initdb scripts are only executed during the container's first execution and will be skipped if `/bitnami/mysql/data/.user_scripts_initialized` exists.

On the other hand, startdb scripts are executed every time the container is started, ignoring the presence of `.user_scripts_initialized` file.

The bitnami/mysql chart does not yet implement a value to mount startdb scripts, similar to `initdbScriptsConfigMap`, and the message `find: '/docker-entrypoint-startdb.d/': No such file or directory` is expected when no startdb are provided, but should not cause any error related to `initdb` scripts.

**Comment by maxsargentdev** [comment]
Fair enough, the error message is a red herring then!

That being said I am still having issues running initialization scripts, specifically these values:

```
    initdbScripts:
      my_init_script.sql: |
        SET PERSIST require_secure_transport=ON;
```

This was working at one point, however if I use this value now something silently failed and there was nothing useful in the logs.

Let me attempt to repro now and post some evidence.

**Comment by SyedMHashim** [comment]
@migruiz4 I see that in the statefulset template for both primary and secondary there is no volume mount for `/docker-entrypoint-startdb.d`. Can we please have a helm value to configure startdb scripts similar to the `initdbScripts` and subsequently add a volume and volumeMount in the statefulsets for this? This would ensure that the `startdbScripts` are run on every container restart or helm upgrade. I would be more than happy to work on this and submit a PR.

**Comment by migruiz4** [comment]
Hi @SyedMHashim,

It would be great if you could submit a PR. In this link, you can find our [Contributing Guidelines](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md).

If you need any further help with the new feature don't hesitate to let me know.

**Comment by SyedMHashim** [comment]
Hi @migruiz4. I have created the PR. Would appreciate any feedback.

**Comment by carrodher** [comment]
Thank you for submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!

**Comment by doznap** [comment]
Rad! thanks everyone! by the way. Even though the container was throwing that error, it still applied the .mysql as expected. sooo i dunno what its doing but it is functional

**Comment by maxsargentdev** [comment]
@doznap I will test tomorrow with my example and verify we start as correctly and guarantee encryption.

At that point it's probably good to close this.

**Comment by maxsargentdev** [comment]
My installation is working correctly now also.
